views/todo/create2.ejs

```diff
@@ -0,0 +1,24 @@
+<div>
+    <h2>Create Todo</h2>
+
+    <form method="POST" action="/user/create">
```

Comment:
ok so i think adding a todo will be a put request (and not a post) so we need the methodOverride here!